cocotb/_build_libs.py

```diff
@@ -174,6 +170,8 @@ def _get_python_lib():
 
     if os.name == "nt":
         python_lib = _get_python_lib_link() + "." + _get_lib_ext_name()
+    elif sys.platform == "darwin":
+        python_lib = "lib" + _get_python_lib_link() + ".dylib"
```

Comment:
You could just extend ``_get_lib_ext_name()`` instead, right?

Reply:
Yes, I could do that. But I found a few places where the default extension for libraries was assumed to be .so (Linux, Mac) or .dll (Windows). I wasn't sure I caught them all.